.github/workflows/review.yml

```diff
@@ -112,7 +120,7 @@ jobs:
           registry: ${{ secrets.GOOGLE_REGION }}-docker.pkg.dev
           username: ${{ steps.gcloud.outputs.username }}
           password: ${{ steps.gcloud.outputs.password }}
-          tags: "${{ env.SHORT_SHA }}"
+          tags: "${{ env.TAGS }}"
```

Comment:
Does this mean that every container get all tags on the commit?

Reply:
please have a look at our tagging structure (as it is defined in the issue as well):

https://github.com/Rpc-h/RPCh/tags

because of the mono repo we do have multiple tags on the same commit targeting different applications.
What we would like to have is the respective tag on the respective application container.
E.g.
`rpc-server-v0.3.0` on the rpc-server container
This container should not contain the other tags of the commit.